pom.xml

```diff
@@ -90,10 +98,32 @@
             <artifactId>dropwizard-core</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.elasticsearch</groupId>
-            <artifactId>elasticsearch</artifactId>
+            <groupId>org.elasticsearch.client</groupId>
+            <artifactId>transport</artifactId>
+            <version>${elasticsearch.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>commons-logging</groupId>
+                    <artifactId>commons-logging</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>org.elasticsearch.plugin</groupId>
+            <artifactId>transport-netty4-client</artifactId>
             <version>${elasticsearch.version}</version>
         </dependency>
+        <!-- Required for ES 5.x -->
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
```

Comment:
Instead of pulling in a second logging framework, this project should use the [Log4j to SLF4J adapter](https://logging.apache.org/log4j/2.0/log4j-to-slf4j/index.html).